io/net/scp_ssh.py.data/scp_ssh.yaml

```diff
@@ -0,0 +1,3 @@
+Parameters:
+    peerip: "9.40.195.85"
```

Comment:
@prudmirya @abdhaleegit means to say remove the IP from yaml file when upstreaming the code